kafka-ui-react-app/src/components/common/SmartTable/TableRow.tsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { propertyLookup } from 'lib/propertyLookup';
 import { TableState } from 'lib/hooks/useTableState';
+import { Td } from 'components/Topics/List/List.styled';
```

Comment:
This place is a general component importing something from a custom component can be wrong , for the overflow thing just create a class name put the styling in it and put in the <td>
as well as the classname passed with props. this way the component  will be general without custom imports.